bin/lcov

```diff
@@ -4291,6 +4296,17 @@ sub print_overall_rate($$$$$$$$$)
 	info("  branches...: %s\n",
 	     get_overall_line($br_found, $br_hit, "branch", "branches"))
 		if ($br_do);
+
+	if ($ln_do && $fail_under_lines > 0) {
+		my $actual_rate = ($ln_hit / $ln_found);
```

Comment:
For robustness please also handle the case of `$ln_found == 0`. This cannot happen for `--summary`, but e.g. for `--remove`.  Putting the check in here now will make the code also work if this function is extended to other operations in the future.

Reply:
Makes sense, I did that now.